cpg_workflows/inputs.py

```diff
@@ -26,20 +26,41 @@ def create_cohort() -> Cohort:
     """
     Add datasets in the cohort. There exists only one cohort for the workflow run.
     """
-    analysis_dataset_name = get_config()['workflow']['dataset']
-    dataset_names = get_config()['workflow'].get('input_datasets', [analysis_dataset_name])
-    skip_datasets = get_config()['workflow'].get('skip_datasets', [])
+    config = get_config()['workflow']
+    analysis_dataset_name = config['dataset']
+    custom_cohort_ids = config.get('input_cohorts', [])
+    input_datasets = config.get('input_datasets', [])
+    skip_datasets = config.get('skip_datasets', [])
```

Comment:
```suggestion
    from cpg_utils.config import config_retrieve
    
    config = config_retrieve(['workflow'])
    analysis_dataset_name = config_retrieve(['workflow', 'dataset'])
    custom_cohort_ids = config_retrieve(['workflow', 'input_cohorts'], [])
    input_datasets = config_retrieve(['workflow', 'input_datasets'], [])
    skip_datasets = config_retrieve(['workflow', 'skip_datasets'], [])
```

preference for the use of `config_retrieve` instead of `get_config()[key][...]`. `config_retrieve` will fail descriptively if the nested keys aren't present, and incorporates a default value return.